src/NUnitFramework/tests/TestUtilities/Comparers/ObjectToStringComparer.cs

```diff
@@ -37,10 +37,8 @@ public class ObjectToStringComparer : System.Collections.IComparer
         int System.Collections.IComparer.Compare(object x, object y)
         {
             WasCalled = true;
-            int intX = 0;
-            int intY = 0;
 
-            if(int.TryParse(x.ToString(), out intX) && int.TryParse(y.ToString(), out intY))
+            if (int.TryParse(x.ToString(), out var intX) && int.TryParse(y.ToString(), out var intY))

```

Comment:
I'd use `int` here rather than `var`, but I don't think you have made it any less clear WRT the use of `out` either here or anywhere else.

IMO the high level of discussion comes from the fact that by making a global stylistic change you __appear__ to be defining a new standard of how to code, even though it isn't actually proposed as a standard and (again IMO) shouldn't be. Just a comment.

Reply:
`var` back to `int`: do you think it's worth a new PR? Happy to do that.

I'm glad you're making that distinction. This is not a standard. Along similar lines, this way future writers seeing this code nearby would hopefully conclude that they now have either option open to them rather than just the old way.

Reply:
I don't think it's worth it, personally, but up to you.